### ISSUE TITLE
`e.localeCompare is not a function` error when trying to display quality profiles modal

### ISSUE THREAD

**Issue by demize** [Sonarr/Sonarr#6985]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When I try to add or edit a quality profile, the modal displays an error instead: `e.localeCompare is not a function` (full stack trace provided below).

### Expected Behavior

The modal will display the correct form and information.

### Steps To Reproduce

On the current develop version published by the LSIO docker container, try to open or add a quality profile. You'll be greeted with this error. 

### Environment

```markdown
- OS: Unraid
- Sonarr: 4.0.8.1893
- Docker Install: Yes, LSIO container
- Using Reverse Proxy: Yes, Unraid haproxy
- Browser: Firefox 128 on Gentoo; Safari on iOS 18
- Database: Sqlite 3.45.3
```

### What branch are you running?

Develop

### Trace Logs?

```
2024-07-18 11:55:39.2|Trace|DiskProviderBase|Deleting file: /config/logs/sonarr.debug.txt
2024-07-18 11:55:39.2|Trace|CommandQueueManager|Updating command status
2024-07-18 11:55:39.2|Trace|EventAggregator|Publishing CommandUpdatedEvent
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2024-07-18 11:55:39.2|Trace|EventAggregator|Publishing CommandExecutedEvent
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandExecutedEvent -> TaskManager
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandExecutedEvent <- TaskManager
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandExecutedEvent -> TaskController
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandExecutedEvent <- TaskController
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandExecutedEvent -> QualityDefinitionController
2024-07-18 11:55:39.2|Trace|EventAggregator|CommandExecutedEvent <- QualityDefinitionController
2024-07-18 11:55:39.2|Trace|CommandExecutor|DeleteLogFilesCommand <- DeleteLogFilesService [00:00:00.0090277]
2024-07-18 11:55:39.2|Trace|Http|Req: 4188 [GET] /api/v3/command (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:39.2|Trace|CommandQueueManager|Getting all commands
2024-07-18 11:55:39.2|Trace|Http|Res: 4188 [GET] /api/v3/command: 200.OK (2 ms)
2024-07-18 11:55:39.2|Debug|Api|[GET] /api/v3/command: 200.OK (2 ms)
2024-07-18 11:55:39.3|Trace|Http|Req: 4189 [GET] /api/v3/log/file (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:39.3|Trace|Http|Res: 4189 [GET] /api/v3/log/file: 200.OK (2 ms)
2024-07-18 11:55:39.3|Debug|Api|[GET] /api/v3/log/file: 200.OK (2 ms)
2024-07-18 11:55:40.9|Trace|Http|Res: 4086 [GET] /signalr/messages?access_token=(removed)&id=IpRIstdYs042Z9zBORnoEQ: 101.SwitchingProtocols (154866 ms)
2024-07-18 11:55:40.9|Trace|Http|Req: 4190 [GET] / (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:40.9|Trace|Http|Res: 4190 [GET] /: 200.OK (1 ms)
2024-07-18 11:55:41.0|Trace|Http|Req: 4191 [POST] /signalr/messages/negotiate?access_token=(removed)&negotiateVersion=1 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.0|Trace|Http|Res: 4191 [POST] /signalr/messages/negotiate?access_token=(removed)&negotiateVersion=1: 200.OK (1 ms)
2024-07-18 11:55:41.0|Trace|Http|Req: 4192 [GET] /initialize.json?t=1721318141032 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.0|Trace|Http|Res: 4192 [GET] /initialize.json?t=1721318141032: 200.OK (1 ms)
2024-07-18 11:55:41.1|Trace|Http|Req: 4193 [GET] /api/v3/localization/language (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.1|Trace|ConfigService|Using default config value for 'uilanguage' defaultValue:'1'
2024-07-18 11:55:41.1|Trace|Http|Res: 4193 [GET] /api/v3/localization/language: 200.OK (2 ms)
2024-07-18 11:55:41.1|Debug|Api|[GET] /api/v3/localization/language: 200.OK (2 ms)
2024-07-18 11:55:41.2|Trace|Http|Req: 4194 [GET] /api/v3/series (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4195 [GET] /api/v3/customFilter (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Res: 4195 [GET] /api/v3/customFilter: 200.OK (1 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/customFilter: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|Http|Req: 4196 [GET] /api/v3/tag (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4197 [GET] /api/v3/qualityprofile (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Res: 4196 [GET] /api/v3/tag: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|Http|Req: 4198 [GET] /api/v3/language (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4199 [GET] /api/v3/importlist (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4200 [GET] /api/v3/indexerFlag (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4201 [GET] /api/v3/config/ui (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4202 [GET] /api/v3/localization (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Req: 4203 [GET] /api/v3/system/status (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.2|Trace|Http|Res: 4197 [GET] /api/v3/qualityprofile: 200.OK (1 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/tag: 200.OK (1 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/qualityprofile: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|Http|Res: 4199 [GET] /api/v3/importlist: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|Http|Res: 4200 [GET] /api/v3/indexerFlag: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'firstdayofweek' defaultValue:'0'
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'uilanguage' defaultValue:'1'
2024-07-18 11:55:41.2|Trace|Http|Res: 4203 [GET] /api/v3/system/status: 200.OK (3 ms)
2024-07-18 11:55:41.2|Trace|Http|Res: 4198 [GET] /api/v3/language: 200.OK (1 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/system/status: 200.OK (3 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/importlist: 200.OK (1 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/indexerFlag: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'calendarweekcolumnheader' defaultValue:'ddd M/D'
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/language: 200.OK (1 ms)
2024-07-18 11:55:41.2|Trace|Http|Res: 4202 [GET] /api/v3/localization: 200.OK (8 ms)
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'shortdateformat' defaultValue:'MMM D YYYY'
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/localization: 200.OK (8 ms)
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'longdateformat' defaultValue:'dddd, MMMM D YYYY'
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'timeformat' defaultValue:'h(:mm)a'
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'showrelativedates' defaultValue:'True'
2024-07-18 11:55:41.2|Trace|ConfigService|Using default config value for 'uilanguage' defaultValue:'1'
2024-07-18 11:55:41.2|Trace|Http|Res: 4201 [GET] /api/v3/config/ui: 200.OK (13 ms)
2024-07-18 11:55:41.2|Debug|Api|[GET] /api/v3/config/ui: 200.OK (13 ms)
2024-07-18 11:55:41.5|Trace|Http|Res: 4194 [GET] /api/v3/series: 200.OK (397 ms)
2024-07-18 11:55:41.5|Debug|Api|[GET] /api/v3/series: 200.OK (397 ms)
2024-07-18 11:55:41.6|Trace|Http|Req: 4204 [POST] /signalr/messages/negotiate?access_token=(removed)&negotiateVersion=1 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.6|Trace|Http|Req: 4205 [GET] /api/v3/queue/status (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.6|Trace|Http|Req: 4206 [GET] /api/v3/health (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.6|Trace|Http|Req: 4207 [GET] /api/v3/queue/details?all=true (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.6|Trace|Http|Req: 4208 [GET] /api/v3/series (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.6|Trace|Http|Res: 4204 [POST] /signalr/messages/negotiate?access_token=(removed)&negotiateVersion=1: 200.OK (1 ms)
2024-07-18 11:55:41.6|Trace|Http|Res: 4205 [GET] /api/v3/queue/status: 200.OK (2 ms)
2024-07-18 11:55:41.6|Trace|Http|Res: 4206 [GET] /api/v3/health: 200.OK (2 ms)
2024-07-18 11:55:41.6|Trace|Http|Res: 4207 [GET] /api/v3/queue/details?all=true: 200.OK (3 ms)
2024-07-18 11:55:41.6|Debug|Api|[GET] /api/v3/queue/status: 200.OK (2 ms)
2024-07-18 11:55:41.6|Debug|Api|[GET] /api/v3/health: 200.OK (2 ms)
2024-07-18 11:55:41.6|Debug|Api|[GET] /api/v3/queue/details?all=true: 200.OK (3 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4209 [GET] /MediaCover/127/poster-500.jpg?lastWrite=638414055369360653 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4209 [GET] /MediaCover/127/poster-500.jpg?lastWrite=638414055369360653: 200.OK (1 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4210 [GET] /MediaCover/138/poster-500.jpg?lastWrite=638477454703720751 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4210 [GET] /MediaCover/138/poster-500.jpg?lastWrite=638477454703720751: 200.OK (1 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4211 [GET] /signalr/messages?access_token=(removed)&id=fyHlMgD6ybLJFzmTEfZurw (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4213 [GET] /MediaCover/153/poster-500.jpg?lastWrite=638490861334791303 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4212 [GET] /MediaCover/36/poster-500.jpg?lastWrite=638220896539416223 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4214 [GET] /MediaCover/161/poster-500.jpg?lastWrite=638474442260223137 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4215 [GET] /MediaCover/187/poster-500.jpg?lastWrite=638558252453073044 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4216 [GET] /MediaCover/151/poster-500.jpg?lastWrite=638547948788535253 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4217 [GET] /MediaCover/103/poster-500.jpg?lastWrite=638376662590389698 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4218 [GET] /MediaCover/117/poster-500.jpg?lastWrite=638400132549079776 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4219 [GET] /MediaCover/45/poster-500.jpg?lastWrite=638244418748138856 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4220 [GET] /MediaCover/135/poster-500.jpg?lastWrite=638433626482329891 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4213 [GET] /MediaCover/153/poster-500.jpg?lastWrite=638490861334791303: 200.OK (4 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4221 [GET] /MediaCover/67/poster-500.jpg?lastWrite=638310413015899120 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4220 [GET] /MediaCover/135/poster-500.jpg?lastWrite=638433626482329891: 200.OK (4 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4222 [GET] /MediaCover/124/poster-500.jpg?lastWrite=638406949772798282 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4223 [GET] /MediaCover/185/poster-500.jpg?lastWrite=638554823955258714 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4224 [GET] /MediaCover/120/poster-500.jpg?lastWrite=638405785554223201 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4215 [GET] /MediaCover/187/poster-500.jpg?lastWrite=638558252453073044: 200.OK (5 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4216 [GET] /MediaCover/151/poster-500.jpg?lastWrite=638547948788535253: 200.OK (5 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4225 [GET] /MediaCover/83/poster-500.jpg?lastWrite=638339585805329085 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4218 [GET] /MediaCover/117/poster-500.jpg?lastWrite=638400132549079776: 200.OK (3 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4226 [GET] /MediaCover/71/poster-500.jpg?lastWrite=638322234553681107 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4227 [GET] /MediaCover/41/poster-500.jpg?lastWrite=638232430112150915 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4212 [GET] /MediaCover/36/poster-500.jpg?lastWrite=638220896539416223: 200.OK (9 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4214 [GET] /MediaCover/161/poster-500.jpg?lastWrite=638474442260223137: 200.OK (11 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4217 [GET] /MediaCover/103/poster-500.jpg?lastWrite=638376662590389698: 200.OK (9 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4223 [GET] /MediaCover/185/poster-500.jpg?lastWrite=638554823955258714: 200.OK (6 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4228 [GET] /MediaCover/60/poster-500.jpg?lastWrite=638257394279721582 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4229 [GET] /MediaCover/81/poster-500.jpg?lastWrite=638329704861493279 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4230 [GET] /MediaCover/3/poster-500.jpg?lastWrite=638101107952333839 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4222 [GET] /MediaCover/124/poster-500.jpg?lastWrite=638406949772798282: 200.OK (14 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4232 [GET] /MediaCover/188/poster-500.jpg?lastWrite=638558252763824119 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4233 [GET] /MediaCover/57/poster-500.jpg?lastWrite=638256664911073006 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4234 [GET] /MediaCover/128/poster-500.jpg?lastWrite=638419148532660757 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4235 [GET] /MediaCover/175/poster-500.jpg?lastWrite=638517543109233799 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Req: 4236 [GET] /MediaCover/105/poster-500.jpg?lastWrite=638377813848309379 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4219 [GET] /MediaCover/45/poster-500.jpg?lastWrite=638244418748138856: 200.OK (12 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4221 [GET] /MediaCover/67/poster-500.jpg?lastWrite=638310413015899120: 200.OK (12 ms)
2024-07-18 11:55:41.9|Trace|Http|Req: 4231 [GET] /MediaCover/73/poster-500.jpg?lastWrite=638323230750059581 (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:41.9|Trace|Http|Res: 4229 [GET] /MediaCover/81/poster-500.jpg?lastWrite=638329704861493279: 200.OK (6 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4233 [GET] /MediaCover/57/poster-500.jpg?lastWrite=638256664911073006: 200.OK (6 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4228 [GET] /MediaCover/60/poster-500.jpg?lastWrite=638257394279721582: 200.OK (10 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4236 [GET] /MediaCover/105/poster-500.jpg?lastWrite=638377813848309379: 200.OK (7 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4230 [GET] /MediaCover/3/poster-500.jpg?lastWrite=638101107952333839: 200.OK (10 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4232 [GET] /MediaCover/188/poster-500.jpg?lastWrite=638558252763824119: 200.OK (9 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4234 [GET] /MediaCover/128/poster-500.jpg?lastWrite=638419148532660757: 200.OK (9 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4235 [GET] /MediaCover/175/poster-500.jpg?lastWrite=638517543109233799: 200.OK (9 ms)
2024-07-18 11:55:41.9|Trace|Http|Res: 4231 [GET] /MediaCover/73/poster-500.jpg?lastWrite=638323230750059581: 200.OK (11 ms)
2024-07-18 11:55:42.0|Trace|Http|Res: 4208 [GET] /api/v3/series: 200.OK (329 ms)
2024-07-18 11:55:42.0|Debug|Api|[GET] /api/v3/series: 200.OK (329 ms)
2024-07-18 11:55:42.0|Trace|Http|Res: 4227 [GET] /MediaCover/41/poster-500.jpg?lastWrite=638232430112150915: 200.OK (43 ms)
2024-07-18 11:55:42.0|Trace|Http|Res: 4226 [GET] /MediaCover/71/poster-500.jpg?lastWrite=638322234553681107: 200.OK (43 ms)
2024-07-18 11:55:42.0|Trace|Http|Res: 4224 [GET] /MediaCover/120/poster-500.jpg?lastWrite=638405785554223201: 200.OK (46 ms)
2024-07-18 11:55:42.0|Trace|Http|Res: 4225 [GET] /MediaCover/83/poster-500.jpg?lastWrite=638339585805329085: 200.OK (44 ms)
2024-07-18 11:55:46.1|Trace|Http|Req: 4237 [GET] /api/v3/qualityprofile (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:46.1|Trace|Http|Res: 4237 [GET] /api/v3/qualityprofile: 200.OK (1 ms)
2024-07-18 11:55:46.1|Debug|Api|[GET] /api/v3/qualityprofile: 200.OK (1 ms)
2024-07-18 11:55:46.1|Trace|Http|Req: 4238 [GET] /api/v3/delayprofile (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:46.1|Trace|Http|Req: 4239 [GET] /api/v3/releaseprofile (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:46.1|Trace|Http|Req: 4240 [GET] /api/v3/indexer (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:46.1|Trace|Http|Res: 4238 [GET] /api/v3/delayprofile: 200.OK (1 ms)
2024-07-18 11:55:46.1|Trace|Http|Res: 4239 [GET] /api/v3/releaseprofile: 200.OK (1 ms)
2024-07-18 11:55:46.1|Debug|Api|[GET] /api/v3/delayprofile: 200.OK (1 ms)
2024-07-18 11:55:46.1|Debug|Api|[GET] /api/v3/releaseprofile: 200.OK (1 ms)
2024-07-18 11:55:46.1|Trace|Http|Res: 4240 [GET] /api/v3/indexer: 200.OK (5 ms)
2024-07-18 11:55:46.1|Debug|Api|[GET] /api/v3/indexer: 200.OK (5 ms)
2024-07-18 11:55:50.8|Trace|Http|Req: 4241 [GET] /api/v3/health (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:50.8|Trace|Http|Req: 4242 [GET] /api/v3/diskspace (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:50.8|Trace|Http|Req: 4243 [GET] /api/v3/system/status (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:50.8|Trace|Http|Res: 4241 [GET] /api/v3/health: 200.OK (1 ms)
2024-07-18 11:55:50.8|Trace|Http|Res: 4243 [GET] /api/v3/system/status: 200.OK (2 ms)
2024-07-18 11:55:50.8|Debug|Api|[GET] /api/v3/health: 200.OK (1 ms)
2024-07-18 11:55:50.8|Debug|Api|[GET] /api/v3/system/status: 200.OK (2 ms)
2024-07-18 11:55:50.8|Trace|Http|Res: 4242 [GET] /api/v3/diskspace: 200.OK (29 ms)
2024-07-18 11:55:50.8|Debug|Api|[GET] /api/v3/diskspace: 200.OK (29 ms)
2024-07-18 11:55:51.9|Trace|Http|Req: 4244 [GET] /api/v3/log/file (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
2024-07-18 11:55:51.9|Trace|Http|Res: 4244 [GET] /api/v3/log/file: 200.OK (1 ms)
2024-07-18 11:55:51.9|Debug|Api|[GET] /api/v3/log/file: 200.OK (1 ms)
2024-07-18 11:55:53.2|Trace|Http|Req: 4245 [GET] /logfile/sonarr.trace.txt (from 10.95.95.220 Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0)
```

### Anything else?

Stack trace:

```
e.localeCompare is not a function
  at profileFormatItems].sort( (webpack://sonarr/frontend/src/Settings/Profiles/Quality/QualityProfileFormatItems.js:24:13)
  at calcOrder (webpack://sonarr/frontend/src/Settings/Profiles/Quality/QualityProfileFormatItems.js:19:33)
  at constructor (webpack://sonarr/frontend/src/Settings/Profiles/Quality/QualityProfileFormatItems.js:37:13)
  at Mg (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:135:171)
  at pi (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:181:145)
  at a=b.pendingProps;f=e._init;e=f(e._payload);b.type=e;f=b.tag=hk(e);a=lg(e,a);switch (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:269:426)
  at Pj (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:250:349)
  at Pj (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:250:280)
  at Pj (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:250:137)
  at Lj (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:243:162)
  at jg (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:123:114)
  at unstable_next (webpack://sonarr/node_modules/scheduler/cjs/scheduler.production.min.js:18:342)
  at eg (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:122:324)
  at jg (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:123:63)
  at eg (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:122:427)
  at Hb (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:292:100)
  at $c (webpack://sonarr/node_modules/react-dom/cjs/react-dom.production.min.js:73:354)
  at Array.prototype.slice.call (webpack://sonarr/node_modules/@sentry/browser/esm/helpers.js:91:16)
```

My other Sonarr instance does not have this issue, running the Hotio docker container on main (4.0.8.1874).

**Comment by thezoggy** [comment]
already fixed in newer code, update

**Comment by demize** [comment]
> already fixed in newer code, update - for develop: 4.0.8.1902

Cool, good to know; I'll close this myself then :)
